integrations/twitter-v2.js

```diff
@@ -0,0 +1,88 @@
+try {
```

Comment:
wait what?
Why did I read all of the v1 then? :smiling_face_with_tear: 

Stopping here, gotta sleep. But what should I read?

Reply:
Twitter has two API versions. I probably don't need to migrate to v2 yet, but I started there before I realized that API doesn't yet support DMs. I'm fetching hashtag tweets from API v1, DMs from API v2.

Reply:
Oh OK. TLDR: twitter API sucks? :grin:
Raises the question if we need to reflect that on our side. Maybe just `integration/twitter` and some functions use the `v1` and others the `v2`? I'll know more after reading this file.

Reply:
Kk. Another option is to just use V1 exclusively. I'm sure they're not going to deprecate it any time soon if V2 isn't feature complete.